libs/gl-client-py/glclient/__init__.py

```diff
@@ -55,11 +55,12 @@ def is_running(self) -> bool:
 
 class Scheduler(object):
 
-    def __init__(self, node_id: bytes, network: str, tls: TlsConfig):
+    def __init__(self, node_id: bytes, network: str, tls: TlsConfig, rune: str=""):
```

Comment:
Setting a default does not sounds like a good idea, as it suggests that users don't need to set this, and they'll run into issues once we start enforcing these constraints.

We should rather start consolidating all of these arguments into a single blob that can be upgraded from version to version. A protobuf struct could serve for this: we hand out a blob, and expect it to be handed back when loading, so the callers just need to persist however they want and not care about the details.

Reply:
Yes I already had this in mind as a next step in a separate PR. I could also add it in this PR if you think it is more convenient to have these changes tied together so that people only need to `recover` once.
In this case I can also add these changes to the `cli` tool with a descriptive info/error message.